On Python 3.12 two more tests fail for me.

```bash
[...]
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/test_sync.py::test_diff_should_not_uninstall - AssertionError: assert {'distribute'...ols', 'wheel'} == set()
FAILED tests/test_cli_compile.py::test_combine_different_extras_of_the_same_package[backtracking resolver] - AssertionError: assert 'fake-colorfu...irements.in\n' == 'fake-colorfu...ir...
=========== 2 failed, 723 passed, 333 warnings in 104.22s (0:01:44) ============
``

--- a/pkgs/development/python-modules/pip-tools/default.nix
+++ b/pkgs/development/python-modules/pip-tools/default.nix
@@ -66,6 +66,8 @@ buildPythonPackage rec {
     "test_bad_setup_file"
     # Assertion error
     "test_compile_recursive_extras"
+    "test_combine_different_extras_of_the_same_package"
+    "test_diff_should_not_uninstall"
   ];
 
   pythonImportsCheck = [

